tests/operator_test.go

```diff
@@ -328,7 +328,7 @@ var _ = Describe("ALL Operator tests", func() {
 					Operator: corev1.TolerationOpExists,
```

Comment:
How about adding some tests that modify the new replica count fields and verify the results?

Reply:
Hi, could you please explain the meaning of 'test_id'? Is it possible for me to set my test to 4785?

Reply:
Just have to be unique that number is fine!